src/components/CTypeOverview/CTypeOverview.astro

```diff
@@ -16,7 +16,7 @@ const { title, id, description, attestationsCreated, tags } =
 <li class=`${containerStyles.container} container`>
   <h2>
     <a href={generatePath(paths.ctypeDetails, id)} class="title">
-      {title}
+      {title.length > 0 ? title : '∅'}
```

Comment:
It’s unclear to me whether this is still needed, but okay. This could also be spelled simpler by the way.
```suggestion
      {title || '∅'}
```

Reply:
I thought `""` was _truthty_, but you are right it is _falsy_, so that's possible.